benchmarks/push_results.py

```diff
@@ -3,7 +3,7 @@
 
 import pandas as pd
 from huggingface_hub import hf_hub_download, upload_file
-from huggingface_hub.utils._errors import EntryNotFoundError
+from huggingface_hub.utils import EntryNotFoundError
```

Comment:
```suggestion
from huggingface_hub.errors import EntryNotFoundError
```

This should be even more future-proof (though it's already fine :) ).

Reply:
It requires a library-wide change and I will prefer a separate PR for that.